addons/auth_totp/data/mail_templates.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="mail_template_totp_invite" model="mail.template">
+        <field name="name">TOTP on User: Invite by email</field>
+        <field name="model_id" ref="base.model_res_users" />
+        <field name="subject">Invitation to activate 2-Step authentication on your Odoo account</field>
+        <field name="email_to">${(object.partner_id.email_formatted or object.email) |safe}</field>
+        <field name="partner_to">${object.partner_id.id}</field>
+        <field name="lang">${object.partner_id.lang}</field>
+        <field name="body_html" type="html">
+            <div style="margin: 0px; padding: 0px; font-size: 13px;">
```

Comment:
Just to save space, you may left-indent html (begin you div at column 0), so that editor does not explode.